templates/default/sinatra_app.conf.erb

```diff
@@ -8,9 +8,10 @@
     CustomLog <%= @node[:apache][:log_dir] %>/access-<%= @params[:server_name] %>.log combined
     
     <Directory <%= "#{@params[:docroot]}/public" %>>
-        Options FollowSymLinks -MultiViews
+        Options +FollowSymLinks -MultiViews
```

Comment:
Without this apache's error log will tell you that all the options have to start with +/- or none of them can.